pandas/tests/interchange/test_impl.py

```diff
@@ -14,7 +14,7 @@
     ColumnNullType,
     DtypeKind,
 )
-from pandas.core.interchange.from_dataframe import from_dataframe
+from pandas.core.interchange._from_dataframe import from_dataframe
```

Comment:
does it work to make this just `from pandas.core.interchange import from_dataframe`?

Reply:
also, can you run the pre-commit checks please? check the contributing guide for how to run them

Reply:
Greetings @MarcoGorelli, i added a new commit in my repo and i have passed all the tests this time (i fixed the problem which was the isort for the imports). As for your first question i tried that but it would not work. Feel free to give me some feedback!